src/hotspot/share/runtime/globals.hpp

```diff
@@ -366,6 +366,9 @@ const int ObjectAlignmentInBytes = 8;
   product(bool, UseSignumIntrinsic, false, DIAGNOSTIC,                      \
           "Enables intrinsification of Math.signum")                        \
                                                                             \
+  product_pd(bool, MoveIntrinsicStubsGen, DIAGNOSTIC,                       \
```

Comment:
The flag name looks confusing. What about `LazyCompilerStubGeneration` or even `LazyStubGeneration`?

Reply:
I know, I am struggling with this name too. But I can't find better one.
`LazyStubGeneration` is reserved for an other RFE: [JDK-8304422](https://bugs.openjdk.org/browse/JDK-8304422).
And it is not lazy. Compiler stubs generation is **delayed** until Compiler runtime initialization but they all still generated during initialization phase.
How about `DelayCompilerStubsGeneration` ?

Reply:
I think `DelayCompilerStubsGeneration` sounds OK

Reply:
`DelayCompilerStubsGeneration` sounds good.

Reply:
Okay.